frontend/components/elements/StoryItem.vue

```diff
@@ -110,7 +132,7 @@
                     size="sm" 
                     variant="danger" 
                     title="Delete" 
-                    v-confirm="remove">
+                    v-confirm="{ action: remove, text: 'Are you sure you want to delete this story?' }">
```

Comment:
This is a trap for me. Once I want to delete a story - I cannot get away from the alert. I need to restart the browser.

Reply:
I can't reproduce this - which browser are you using?

Reply:
newest Chrome with Linux

Reply:
But Firefox has the same behaviour.

Reply:
hmmmmm. does it work with the other delete buttons? courses, topics etc? if it's just the stories I'd say let's leave it for now and check back if it works once the pages are refactored.

Reply:
It is just the story deletion. Fun Fact: Deletion works after exactly confirming 7 times, that I am sure to delete the story :see_no_evil:

Reply:
well you just gotta make sure you're REALLY sure.. I'd say let's see if the problem persists after page redesign and see if we can find a cause then.